handler/src/main/java/io/netty/handler/ssl/OpenSslEngine.java

```diff
@@ -1221,6 +1234,33 @@ public boolean getEnableSessionCreation() {
     }
 
     @Override
+    public SSLParameters getSSLParameters() {
+        SSLParameters sslParameters = super.getSSLParameters();
+
+        int version = PlatformDependent.javaVersion();
+        if (version >= 6) {
```

Comment:
I think `setEndpointIdentificationAlgorithm(..)` was added in Java 7, not Java 6 (just like `setAlgorithmConstrants(..)`).